pkg/registry/controller/etcd/etcd_test.go

```diff
@@ -531,6 +536,86 @@ func TestEtcdWatchControllersMatch(t *testing.T) {
 	watching.Stop()
 }
 
+func TestEtcdWatchControllersFields(t *testing.T) {
+	ctx := api.WithNamespace(api.NewDefaultContext(), validController.Namespace)
+	storage, fakeClient := newStorage(t)
+	fakeClient.ExpectNotFoundGet(etcdgeneric.NamespaceKeyRootFunc(ctx, "/registry/pods"))
+
+	testFieldMap := map[int][]fields.Set{
+		PASS: {
+			map[string]string{"status.replicas": "0"},
```

Comment:
fields.Set instead of map[string]string? ...and if you do that I think `gofmt -s` will be able to omit the type completely.